modules/programs/alot.nix

```
@@ -22,6 +22,9 @@ let
           sendmail_command =
             optionalString (alot.sendMailCommand != null) alot.sendMailCommand;
         }
+        // optionalAttrs (aliases != []) {
+          aliases = builtins.concatStringsSep "," aliases;
```

I don't think you need the `builtins.` here.
```suggestion
          aliases = concatStringsSep "," aliases;
```